gluetool/tests/test_normalize_option.py

```diff
@@ -0,0 +1,83 @@
+import os
+import string
+
+import pytest
+from hypothesis import assume, given, strategies as st
+
+from gluetool.utils import normalize_bool_option, normalize_multistring_option, normalize_path, normalize_path_option
+
+
+def all_casings(input_string):
+    if not input_string:
+        yield ""
+
+    else:
+        first = input_string[:1]
+        if first.lower() == first.upper():
+            for sub_casing in all_casings(input_string[1:]):
+                yield first + sub_casing
+
+        else:
+            for sub_casing in all_casings(input_string[1:]):
+                yield first.lower() + sub_casing
+                yield first.upper() + sub_casing
+
+
+# all of these are expected to evaluate to ``True``
+POSITIVE_BOOLEAM_OPTIONS = [
+    True] \
+    + list(all_casings('yes')) \
+    + list(all_casings('true')) \
+    + ['1'] \
+    + ['y', 'Y'] \
+    + list(all_casings('on'))
+
+
+# test not just values, but also check it works when they are wrapped by some whitespace
+@pytest.mark.parametrize('value',
+                         POSITIVE_BOOLEAM_OPTIONS + ['\t {}\t '.format(value) for value in POSITIVE_BOOLEAM_OPTIONS])
+def test_normalize_bool_positive(value):
+    assert normalize_bool_option(value) is True
+
+
+@given(value=st.one_of(st.just(False), st.text(alphabet=string.printable)))
```

Comment:
What about to add comment for those who are not familiar with this library?

Reply:
Right, added.